compiler/lsp-tests/src/Main.hs

```diff
@@ -435,7 +435,7 @@ executeCommandTests run _ = testGroup "execute command"
         Just escapedFp <- pure $ uriToFilePath (main' ^. uri)
         actualDotString :: ExecuteCommandResponse <- LSP.request WorkspaceExecuteCommand $ ExecuteCommandParams
            "daml/damlVisualize"  (Just (List [Aeson.String $ T.pack escapedFp]))
-        let expectedDotString = "digraph G {\ncompound=true;\nrankdir=LR;\nsubgraph cluster_Coin{\nn0[label=Create][color=green]; \nn1[label=Archive][color=red]; \nn2[label=Delete][color=red]; \nlabel=Coin;color=blue\n}\n}\n"
+        let expectedDotString = "digraph G {\ncompound=true;\nrankdir=LR;\nsubgraph cluster_Coin{\nn0[label=Create][color=green]; \nn1[label=Archive][color=red]; \nn2[label=Delete][color=red]; \nlabel=<<table align = \"left\" border=\"0\" cellborder=\"0\" cellspacing=\"1\">\n<tr><td align=\"center\"><b>Coin</b></td></tr><tr><td align=\"left\">owner</td></tr> \n</table>>;color=blue\n}\n}\n"
```

Comment:
Can we get some proper tests in the shake tests that this works like it is supposed to?

Reply:
Just testing a bunch of different case and making sure (without golden tests) that they all behave as you expect, e.g., records, nested records, variants, …

Reply:
What I am trying to say is that we should have tests that check for this in the shake tests. Given that this change did not change any of the Shake tests, they evidently don’t test your change here so we should extend them to test that fields are correctly represented in the graph.

Reply:
I have updated the tests to assert the fields that were added in this PR. The bit that is not tested in these ShakeIdeTest is the style (the table one which shows template data fields within the DOT file).